src/cloudShellRunner.ts

```diff
@@ -52,50 +52,67 @@ export class CloudShellRunner extends BaseRunner {
 
     protected startCloudShell(playbook: string): void {
         const msgOption: vscode.MessageOptions = { modal: false };
-        const msgItem: vscode.MessageItem = { title: 'Confirm' };
+        const msgItem: vscode.MessageItem = { title: 'Confirm & Not show this again' };
 
```

Comment:
just little thing, i think Not should be lowercase --> "Confirm and don't show this again"
or actually maybe better.

Reply:
i think & is more obvious..

Reply:
ok so just add lowercase "don't"

Reply:
I'll update it "Don't" given it's &